testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js

```diff
@@ -88,6 +88,76 @@ QUnit.test("searchValue from value to empty", function(assert) {
     assert.equal($items.length, 6, "6 items were rendered after filtration");
 });
 
+QUnit.test("search should consider dataSource simple sorting", function(assert) {
```

Comment:
Seems that this test duplicates the dataAdapter test. You can expand that test and check it with a searchValue